core/capabilities/targets/write_target.go

```diff
@@ -340,6 +338,32 @@ func (cap *WriteTarget) Execute(ctx context.Context, rawRequest capabilities.Cap
 				continue
 			}
 			switch txStatus {
+			case commontypes.Unconfirmed:
```

Comment:
Please add a comment above explaining that a TxStatus of Unconfirmed actually means "Confirmed", i.e. the transaction has landed on chain but isn't finalized.